editor/editor_file_system.cpp

```diff
@@ -215,8 +216,63 @@ EditorFileSystemDirectory::~EditorFileSystemDirectory() {
 	}
 }
 
+EditorFileSystem::ScannedDirectory::~ScannedDirectory() {
+	for (int i = 0; i < subdirs.size(); i++) {
+		memdelete(subdirs[i]);
```

Comment:
```suggestion
	for (ScannedDirectory *dir : subdirs) {
		memdelete(dir);
```

Reply:
Done! Did the same thing in `~EditorFileSystemDirectory`  just above.